app/pods/cases/case/subcases/overview/template.hbs

```diff
@@ -2,6 +2,7 @@
 {{subcases/subcases-overview-header
   model=model
   isShowingProcess=isShowingProcess
+  backLink=(action "backLinkAction")
```

Comment:
de action handler wordt deprecated [https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/](https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/)

Kunnen we hier `{{on "click" this.backLinkAction}}` schrijven ookal is het geen octane component?

Reply:
ik geef die action mee naar beneden he. Is geen on click hier